data/component-design-tokens/input-design-tokens.json

```diff
@@ -1,67 +1,70 @@
 {
   "text-input-border": {
-    "value": "inset 0 0 0 {borderWidth.s} {color.greyscale.700}",
+    "value": "inset 0 0 0 {inputs.border-width.default} {inputs.color.border.default}",
     "type": "other"
   },
   "text-input-border-disabled": {
-    "value": "inset 0 0 0 {borderWidth.s} {color.greyscale.200}",
+    "value": "inset 0 0 0 {inputs.border-width.default} {inputs.color.border.disabled}",
     "type": "other"
   },
   "text-input-border-error": {
-    "value": "inset 0 0 0 {borderWidth.m} {color.interactive.error}",
+    "value": "inset 0 0 0 {inputs.border-width.hover} {inputs.color.border.error}",
     "type": "other"
   },
   "text-input-border-focus": {
-    "value": "0 0 0 {borderWidth.l} {color.interactive.focus}",
+    "value": "inset 0 0 0 {inputs.border-width.focus} {inputs.color.border.focus}",
     "type": "other"
   },
   "text-input-border-hover": {
-    "value": "inset 0 0 0 {borderWidth.m} {color.interactive.hover}",
+    "value": "inset 0 0 0 {inputs.border-width.hover} {inputs.color.border.hover}",
     "type": "other"
   },
   "text-input-border-radius": {
-    "value": "{borderRadius.m}",
+    "value": "{inputs.border-radius.input}",
     "type": "borderRadius"
   },
   "text-input-color-bg": {
-    "value": "{color.greyscale.white}",
+    "value": "{inputs.color.background.default}",
     "type": "color"
   },
   "text-input-color-bg-disabled": {
-    "value": "{color.greyscale.100}",
+    "value": "{inputs.color.background.disabled}",
     "type": "color"
   },
   "text-input-color-icon-disabled": {
-    "value": "{color.greyscale.500}",
+    "value": "{inputs.color.text.disabled}",
     "type": "color"
   },
   "text-input-color-text": {
     "value": "{color.text.default}",
     "type": "color"
   },
   "text-input-color-text-disabled": {
-    "value": "{color.text.secondary}",
+    "value": "{inputs.color.text.disabled}",
     "type": "color"
   },
   "text-input-color-text-placeholder": {
-    "value": "{color.text.secondary}",
+    "value": "{inputs.color.text.placeholder}",
     "type": "color"
   },
   "text-input-lt-content-color-bg": {
-    "value": "{color.greyscale.100}",
-    "type": "color"
+    "value": "{color.greyscale.150}",
+    "type": "color",
+    "description": "V2: Updated from greyscale.100 to 150"
```

Comment:
What value do you see in adding a v2 change summary in the description? Who is the audience? For me, I can review the changes in the commit history.